cli/utils/browser.js

```diff
@@ -2,7 +2,7 @@
 'use strict';
 
 const util = require('util');
-const open = require('open');
+const open = require('opn');
```

Comment:
The npm module `open` is deprecated and throws a few critical vulnerability warnings. This package is being maintained and does not have the warnings.